data_steward/bq_utils.py

```diff
@@ -124,6 +124,40 @@ def get_validation_results_dataset_id():
     return dataset_id
 
 
+def is_validation_dataset_id(dataset_id):
+    """
+    Check if  bq_consts.VALIDATION_PREFIX is in the dataset_id
+    :param dataset_id: 
+    :return: a bool indicating whether dataset is a validation_dataset
+    """
+    return bq_consts.VALIDATION_PREFIX in dataset_id
+
+
+def get_latest_validation_dataset_id(project_id):
+    """
+    Get the latest validation_dataset_id based on most recent creationTime. 
+    :param project_id: 
+    :return: the most recent validatioN_dataset_id
+    """
+    dataset_id = os.environ.get(bq_consts.MATCH_DATASET, bq_consts.BLANK)
+    if dataset_id == bq_consts.BLANK:
+        validation_datasets = []
+        for dataset in list_datasets(project_id):
+            dataset_id = dataset['datasetReference']['datasetId']
+            if is_validation_dataset_id(dataset_id):
+                validation_datasets.append(get_dataset(project_id, dataset_id))
+
+        if validation_datasets:
+            validation_datasets = sorted(
+                [(dataset['datasetReference']['datasetId'],
+                  int(dataset['creationTime']))
+                 for dataset in validation_datasets],
+                key=lambda x: x[1],
+                reverse=True)
+            dataset_id = validation_datasets[0][0]
+    return dataset_id
```

Comment:
I think we want to move away from relying on the name of a dataset. For instance, I noticed that a validation set `validation_CDR_20190912` was created probably manually by someone. If you feel strongly about this, I can make the adjustment since this is a temporary solution before we figure out how to link the datasets to each other.